css/styles.css

```diff
@@ -0,0 +1,165 @@
+* {
+  font-family: Merriweather, Source Sans Pro, Oswald;
+  font-weight: lighter;
+}
+
+header {
+  background-color: black;
+  float: left;
+  width: 100%;
+  height: 15vh; 
```

Comment:
@kvelez1  remove the height here, and let the content dictate the containers height. Use margin and padding to adjust positioning to match the comp 🔨